src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -108,4 +108,13 @@ public void setMessages(List<Message> messages) {
     }
     return sentMessages;
   }
+
+  /** Return latest message created */
+  public Message getLatestMessage(){
+    if(messages.size() > 0) {
+      int lastMessage = messages.size()-1;
```

Comment:
I was thinking you might want to name this int here "lastMessageIndex", the name threw me off a little as I was reading through

Reply:
Same suggestion for getLatestUser() and getLatestConversation()

Reply:
Done!